packages/widgets/src/blocks/widget-box/edit.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalUseInnerBlocksProps as useInnerBlocksProps,
+	useBlockProps,
+	InnerBlocks,
+} from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import { HEADING_PLACEHOLDER } from './constants';
+
+const TEMPLATE = [ HEADING_PLACEHOLDER ];
+
+export default function Edit() {
+	const innerBlocksProps = useInnerBlocksProps(
+		{
+			className: 'wp-widget-box__inner-blocks',
+		},
+		{
+			template: TEMPLATE,
```

Comment:
Instead of a template we could introduce a "placeholder" for the block. When first inserted it would have a title field into which you would type the title of your Widget (e.g. `Latest Posts`). Then when you click submit it would auto create the Widget Box with a `core/heading` block already inserted and populated.

Just a thought...